docs/content/en/schemas/v1beta7.json

```diff
@@ -1424,6 +1424,17 @@
       "x-intellij-html-description": "<em>beta</em> describes how to do a build on the local docker daemon and optionally push to a repository."
     },
     "LocalDir": {
+      "properties": {
```

Comment:
Is that related to your PR?

Reply:
Tests were failing with the message "json schema files are not up to date. Please run `make generate-schemas` and commit the changes." So I did that and these were the changes.

Reply:
That's not normal. I'd rebase on top of master

Reply:
Done